medical_family/views/medical_patient_view.xml

```diff
@@ -1,47 +1,46 @@
-                                <field name="country_id"/>
-                                <field name="mobile" data-inputmask="'mask': '9999-9999'"/>
-                                <field name="fax" data-inputmask="'mask': '9999-9999'"/>
-                                <field name="state_id"/>
-                                <field name="has_image"/>
-                                <templates>
-                                    <t t-name="kanban-box">
-                                        <t t-set="color" t-value="kanban_color(record.color.raw_value)"/>
-                                        <div t-att-class="color + (record.title.raw_value == 1 ? ' kanban_color_alert' : '')" style="position: relative">
-                                            <a t-if="! read_only_mode" type="delete" style="position: absolute; right: 0; padding: 4px; diplay: inline-block">X</a>
-                                            <div class="module_vignette">
+            <xpath expr="//notebook">
+                <page string="Family">
+                    <field name="family_member_ids" mode="kanban" domain="[('is_patient', '=', True)]"
+                           context="{'default_contact_id': active_id, 'default_use_parent_address': True, 'default_is_patient': True, 'default_customer': True}}">
+                        <kanban>
+                            <field name="color"/>
+                            <field name="name"/>
+                            <field name="title"/>
+                            <field name="email"/>
+                            <field name="parent_id"/>
+                            <field name="is_company"/>
+                            <field name="function"/>
+                            <field name="phone" data-inputmask="'mask': '9999-9999'"/>
+                            <field name="street"/>
+                            <field name="street2"/>
+                            <field name="zip"/>
+                            <field name="city"/>
+                            <field name="country_id"/>
+                            <field name="mobile" data-inputmask="'mask': '9999-9999'"/>
+                            <field name="fax" data-inputmask="'mask': '9999-9999'"/>
+                            <field name="state_id"/>
+                            <field name="has_image"/>
+                            <templates>
+                                <t t-name="kanban-box">
+                                    <t t-set="color" t-value="kanban_color(record.color.raw_value)"/>
+                                    <div t-att-class="color + (record.title.raw_value == 1 ? ' kanban_color_alert' : '')" style="position: relative">
+                                        <a t-if="! read_only_mode" type="delete" style="position: absolute; right: 0; padding: 4px; diplay: inline-block">X</a>
+                                        <div class="module_vignette">
                                             <a type="open">
```

Comment:
Why is this tab level now different?

The process here should have been `CMD+A` then `TAB`